testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/DockerContainer.java

```diff
@@ -350,7 +350,7 @@ private void copyFileFromContainer(String filename, Path targetPath)
         try {
             return Splitter.on("\n")
                     .omitEmptyStrings()
-                    .splitToList(execCommand("/usr/bin/find", path, "-type", "f", "-print"));
+                    .splitToList(execCommand("test", "-d", path, "&&", "/usr/bin/find", path, "-type", "f", "-print"));
```

Comment:
is it actually executed with a shell?

Reply:
indeed, it is. The use of `String[]` is misleading here (`io.trino.tests.product.launcher.env.DockerContainer#execCommand` signature is wrong), as it suggests totally different semantics

let's switch to using a single String instead.
that would also let us realize we should quote the `path` within shell command

Reply:
per https://github.com/trinodb/trino/pull/7578#issuecomment-818572530 ... it's even worse than that.